### PR TITLE
Generate ext/gmp class synopses from stubs

### PR DESCRIPTION
**Pull request by kocsismate**

### CODE REVIEW

File: reference/gmp/gmp.xml [hunk 1]
```diff
@@ -26,22 +25,20 @@
...
+     <classname>GMP</classname>
+    </ooclass>
 
     <classsynopsisinfo>
      <ooclass>
       <classname>GMP</classname>
      </ooclass>
-     <oointerface>
-      <interfacename>Serializable</interfacename>
-     </oointerface>
```
Comment (saundefined) [inline reference/gmp/gmp.xml:37]: Thanks! Looks good, but it seems `[skip-revcheck]` can not be applied

Comment (kocsismate) [reply reference/gmp/gmp.xml:37]: Ah, yes! I should have spotted this one.

Comment (kocsismate) [reply reference/gmp/gmp.xml:37]: I've just amended the commit message. I'm not sure though why the doc referenced `Serializable` as implemented interface...

Comment (kocsismate) [reply reference/gmp/gmp.xml:37]: I mean, I see that GMP uses a custom serialization/deserialization handler, but I don't know whether it should also implement the already deprecated `Serializable` interface

Comment (cmb69) [reply reference/gmp/gmp.xml:37]:
````
> var_dump(serialize(gmp_init(1)));
string(29) "C:3:"GMP":14:{s:1:"1";a:0:{}}"
````
That "C" looks like the result of Serializable::serialize(), what might explain that erroneous `implements Serializable`.

Comment (kocsismate) [reply reference/gmp/gmp.xml:37]: OK, so can I really get rid of the interface, right? I double checked that GMP didn't previously implemented the interface (explicitly) in former PHP version (`var_dump((new ReflectionClass(gmp_init(1)))->getInterfaceNames());` returns `array()`)

Comment (nikic) [reply reference/gmp/gmp.xml:37]: Yeah, GMP did something odd (my bad I guess ^^) I've implemented the __serialize/__unserialize methods in https://github.com/php/php-src/commit/17ceed9ae785a560631865af86038150d9785121.

Comment (kocsismate) [reply reference/gmp/gmp.xml:37]: Thanks for the correction!

Comment (kocsismate) [reply reference/gmp/gmp.xml:37]: I've just added an initial documentation for the 2 new methods

**Comment by cmb69** [comment]
Thank you!

**Pull request closed without merging**